packages/lucia/src/middleware/index.ts

```diff
@@ -218,6 +218,36 @@ export const qwik = (): Middleware<[QwikRequestEvent]> => {
 	};
 };
 
+type ElysiaContext = {
+	request: Request
+	set: {
+		headers: Record<string, string>
+		status?: number | undefined
+		redirect?: string | undefined
+	}
+}
+export const elysia = (): Middleware<[ElysiaContext]> => {
+	return ({ args }) => {
+		const [{ request, set }] = args
+		const requestContext = {
+			request: {
+				url: request.url,
+				method: request.method,
+				headers: {
+					origin: request.headers.get('Origin'),
+					cookie: request.headers.get('Cookie'),
+					authorization: request.headers.get('Authorization'),
+				},
+			},
+			setCookie: (cookie: Cookie) => {
+				set.headers['Set-Cookie'] = cookie.serialize()
```

Comment:
Do you know if setting `Set-Cookie` overrides the current `Set-Cookie` header if it exists, or does it create a new `Set-Cookie` header?

Reply:
It will override it, should we make it so it appends the cookie at the end?

Reply:
Yes

Reply:
updated code